cache-config/t3c-tail/t3c-tail.go

```diff
@@ -0,0 +1,131 @@
+package main
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"regexp"
+	"time"
+
+	"github.com/apache/trafficcontrol/cache-config/t3cutil"
+	"github.com/apache/trafficcontrol/lib/go-log"
+
+	"github.com/nxadm/tail"
+	"github.com/pborman/getopt/v2"
+)
+
+const AppName = "t3c-tail"
+
+// Version is the application version.
+// This is overwritten by the build with the current project version.
+var Version = "0.4"
+
+// GitRevision is the git revision the application was built from.
+// This is overwritten by the build with the current project version.
+var GitRevision = "nogit"
+
+//default time out is 15 seconds, if not included in json input.
+var timeOutSeconds = 15
```

Comment:
Mind renaming this `defaultTimeoutSeconds`, just so it's clear it's a default, not always the timeout?

Also, the comment needs to start with the variable name for GoDoc, e.g. ` // defaultTimeoutSeconds time out is 15 seconds, if not included in json input.

(Technically unexported symbols aren't used by GoDoc, but we try to keep unexported symbols GoDoc-compliant for ease of reading, and so they're automatically compliant if they're changed to be exported in the future.)

Reply:
this has been renamed to `defaultTimeOutMs` and the comment has been fixed too.